doc/source/upgrading_tokudb_myrocks.rst

```diff
@@ -0,0 +1,85 @@
+.. _upgrading_tokudb_myrocks:
+
+=================================================================================================
+Upgrading from Systems that Use the |TokuDB| or |MyRocks| Storage Engine and Partitioned Tables
+=================================================================================================
+
+Due to the limitation imposed by |MySQL|, it is the storage engine that must
+provide support for partitioning.  |MySQL| 8.0 only provides support for
+partitioned table for the |InnoDB| storage engine.
+
+If you use partitioned tables with the |TokuDB| or |MyRocks| storage engine, the
+upgrade may fail if the native partitioning (provided by the storage engine
+itself) is not enabled.
+
+.. Important:: 
+
+   Starting with Percona Server 8.0.28-18, **the TokuDB storage engine is no longer supported and is removed from the installation packages and not enabled in our binary builds**.
```

Comment:
This statement should be 8.0.26 and deprecation unless you want to wait until the 8.0.28 version is released.

Reply:
Removed